packages/react-app/src/components/dashboard/CumulativeEquityChart.tsx

```
@@ -34,39 +34,43 @@ export function CumulativeEquityChart({ data }: CumulativeEquityChartProps) {
-            />
-          </LineChart>
-        </ResponsiveContainer>
-      </div>
-    </MetricCard>
+    <div className="h-64">
+      <ResponsiveContainer width="100%" height="100%">
+        <AreaChart
+          data={data}
+          margin={{ top: 5, right: 5, left: 5, bottom: 5 }}
+        >
+          <CartesianGrid strokeDasharray="3 3" stroke={colors.cardStroke} opacity={0.7} /> // Enhanced gridlines for clarity
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)